app/src/pages/inside/projectSettingsPageContainer/content/elements/ruleList/draggableRuleList.jsx

```diff
@@ -61,12 +61,12 @@ const DraggableRuleListComponent = ({
 DraggableRuleListComponent.propTypes = {
   ...ruleListPropTypes,
   onRuleDrop: PropTypes.func,
-  dragControlComponent: PropTypes.func,
+  tooltipComponent: PropTypes.func,
```

Comment:
In this top-level component, it's not clear what this tooltip is for.
Can we mention that it is intended for `dragControl`?